<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AFCam Record</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="videoLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>480</width>
      <height>360</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>110</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="initButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>50</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Search cameras</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cameraList">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>50</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="faceLabel">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>140</y>
      <width>210</width>
      <height>20</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="newRecButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>New Record</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>180</y>
      <width>210</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Current patient</string>
    </property>
    <widget class="QLabel" name="labelName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelID">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSurname">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Surname</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelAF">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>AF Type</string>
     </property>
    </widget>
    <widget class="QLabel" name="IDvalue">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>200</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>NA</string>
     </property>
    </widget>
    <widget class="QLabel" name="surnameValue">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>200</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>NA</string>
     </property>
    </widget>
    <widget class="QLabel" name="nameValue">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>200</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>NA</string>
     </property>
    </widget>
    <widget class="QLabel" name="AFValue">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>200</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>NA</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>744</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Camera settings</string>
    </property>
    <addaction name="actionSearch_camera"/>
    <addaction name="actionCamera_setup"/>
   </widget>
   <widget class="QMenu" name="menuSubjects">
    <property name="title">
     <string>Subjects</string>
    </property>
    <addaction name="actionInsert_Data"/>
    <addaction name="actionSearch_subjects"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSubjects"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionInsert_Data">
   <property name="text">
    <string>Insert Data</string>
   </property>
  </action>
  <action name="actionSearch_subjects">
   <property name="text">
    <string>Search subjects</string>
   </property>
  </action>
  <action name="actionSearch_camera">
   <property name="text">
    <string>Search camera</string>
   </property>
  </action>
  <action name="actionCamera_setup">
   <property name="text">
    <string>Camera setup</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
